src/SWP/Component/TemplatesSystem/AUTHORS

```diff
@@ -0,0 +1,5 @@
+Here is an list of MUCH-APPRECIATED CONTRIBUTORS --
+people who have submitted patches, reported bugs, added translations, helped
+answer newbie questions, and generally made SWP/TemplatesSystem that much better:
```

Comment:
I would replace `SWP/TemplatesSystem` with `TemplatesSystem Component`. + you can add yourself to the authors list :)